### PR TITLE
More options list

### PR DESCRIPTION
**Pull request by RamzesUAA**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by TetianaLysak — APPROVED** [review]

File: SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/JourneyPage.tsx [hunk 1]
```diff
@@ -1,11 +1,90 @@
...
+    const myRef = React.useRef<BottomSheet>(null);
+
+    const renderInner = () => (
+        <View style={styles.panel}>
+            <MenuButton text="View profile"></MenuButton>
+            <MenuButton text="Message"></MenuButton>
+        </View>
+    )
+
+    if (props.isOpen == true) {
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/JourneyPage.tsx:22]: please simplify this code: init snap to value in the variable and use ternary operator for it, ex:
let index = props.isOpen ? 1 : 0;
myRef?.current?.snapTo(index )

File: SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/JourneyPage.tsx [hunk 2]
```diff
@@ -1,11 +1,90 @@
...
+                initialSnap={0}
+                renderHeader={renderHeader}
+                enabledInnerScrolling={false}
+                onCloseEnd={() => props.setIsOpen(false)}
+            />
+        </View>
+    );
+};
+
+
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/journey/journey-activity/segment-control-activities/JourneyPage.tsx:49]: extra line

File: SoftServeCarMobile/src/activity/journey/more-options/JourneyPageMoreOptions.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
+// import React from 'react';
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/activity/journey/more-options/JourneyPageMoreOptions.tsx:1]: please add todo with an explanation why code is comented

File: SoftServeCarMobile/src/components/bottom-popup/MenuButton.tsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        setColorSeparator('black');
+    }
+
+    const changeColorToWhite = () => {
+        setColorText('black');
+        setColorButton('white');
+        setColorIcon('black');
+        setColorSeparator('#C1C1C5');
+    }
+
```
Comment (AntonukOM) [inline SoftServeCarMobile/src/components/bottom-popup/MenuButton.tsx:25]: please remove extra lines

**Review by AntonukOM — CHANGES_REQUESTED** [review]

**Review by nrevutska — COMMENTED** [review]
There are 3 code smells in your PR: [https://sonarcloud.io/project/issues?id=ita-social-projects_Car-Front-End&pullRequest=63&resolved=false&types=CODE_SMELL]()
Please, pay attention to these analysis  results

**Pull request closed without merging**